assets/javascripts/tests.js

```diff
@@ -90,9 +90,28 @@ function renderTestName(data, type, row) {
         }
         html += dependencyHtml;
     }
-    if (row.comment_count) {
-        html += ' <a href="/tests/' + row.id + '#comments"><i class="test-label label_comment fa fa-comment" title="' +
-            row.comment_count + (row.comment_count != 1 ? ' comments' : ' comment') + ' available"' + '></i></a>';
+    if (row.comment_data) {
+        var bugs = row.comment_data.bugs;
```

Comment:
You can mark many of the variables `const`, e.g.
```suggestion
        const bugs = row.comment_data.bugs;
```

Reply:
done